### DISCUSSION TITLE
2022/11/27/noip2022-tutor-and-gugugu/

### DISCUSSION THREAD

**Discussion by giscus** [DeCalvin2006/blog#10] [General]
# 2022/11/27/noip2022-tutor-and-gugugu/

Day0 颓了一天。下午刷了两个小时的 Phigros，终于把《光》AP 了。 晚上 🍬 叫我们去开会。考试要戴 N95。 感觉越来越不会考试了……  Day1  在酒店 早上第一个起。 吃完饭在房间里充了会电，下楼在大厅当众打 Phi（雾 然后发现自己方舟 HD 无法 FC  在车上 上车发现没有座位）没法打了。 这次公交车直接差点给我胳膊干脱臼了（不愧是公交车水平  在考场  我在最里面

https://blog.decalvin.tk/2022/11/27/noip2022-tutor-and-gugugu/

**Comment by DeCalvin2006** [comment]
想来想去还是少了些什么。
NOIP 2023 RP++。
cbh lzy lgy CEE 2024 RP++。

**Comment by shiziping** [comment]
VP了一下QwQ
T1 $O(nm)$ $100pts$
T2 $0pts$
T3 骗了个性质分 $10pts$
T4 $O(n^2+nlogn+Q)$ $20pts$
%%%DeCalvin Tql

**Reply by DeCalvin2006** [reply]
Orz您太强了我才100+8pts）

**Comment by DeCalvin2006** [comment]
Upd: 100+0+0+8=108
SD一等线108（
lzy 165 %%%
其他人都炸了（（（
后面大概可能我捞个一等就退役了吧（
挺圆满的。AFO也不错（（（
